Packs/Lumu/Integrations/Lumu/Lumu.py

```diff
@@ -0,0 +1,1080 @@
+                demisto.debug(
+                    f"Ignoring Message ({lumu_event_type_name} - {inc_id}) from Cortex to not create a loop between both "
+                    f"parties"
+                )
+                continue
+
+        event_doc["comment"] = event_doc.get("comment") or "from fetching process"
+
+        event_doc |= {
+            "severity": 2,
+            "status": 1,
+            "mirror_instance": INTEGRATION_INSTANCE,
+            "mirror_id": str(inc_id),
+            "mirror_direction": MIRROR_DIRECTION,
+            "mirror_last_sync": datetime.now().strftime("%Y-%m-%dT%H:%M:%SZ"),
+            "mirror_tags": MIRROR_TAGS,
+        }
+
+        incident = {
+            "name": f'lumu - {event_doc.get("description","")} - {inc_id}',
+            "occurred": event_doc["timestamp"] if "timestamp" in event_doc else today,
+            "dbotMirrorId": str(inc_id),
+            "rawJSON": json.dumps(event_doc),
+        }
+        incidents.append(incident)
+        incidents_id.append(str(inc_id))
+
+    incs_to_cortex = []
+    if incidents_id:
+        cache = get_integration_context()
+        if not cache:
+            cache["cache"] = []
+            cache["lumu_incidentsId"] = []
+
+        incidents_id = list(set(incidents_id))
+        cache["cache"].append(incidents_id)
+
+        for inc in incidents:
+            if inc["dbotMirrorId"] in cache["lumu_incidentsId"]:
+                # skip incident, avoid duplication in cortex
+                # do not add the incident, already exist
+                continue
+            else:
+                incs_to_cortex.append(inc)
+                cache["lumu_incidentsId"].append(inc["dbotMirrorId"])
+        cache["lumu_incidentsId"] = list(set(cache["lumu_incidentsId"]))
+        demisto.debug(
+            f'There are {len(cache["cache"])} events queued ready to process their updates'
+        )
+        set_integration_context(cache)
+    next_run = {"last_fetch": str(next_start_time)}
+
+    return next_run, incs_to_cortex
+
+
+def get_modified_remote_data_command(
+    client: Client, args: Dict[str, str]
+) -> GetModifiedRemoteDataResponse:
+    incidents_id = []
+    try:
+
+        cache = get_integration_context()
+
+        incidents_id = cache["cache"].pop(0)
+        demisto.debug(
+            f'FIFO processing, there are {len(cache["cache"])} events left to process'
+        )
+
+    except IndexError:
+        incidents_id = []
+    finally:
+
+        set_integration_context(cache)
+        return GetModifiedRemoteDataResponse(incidents_id)
+
+
+def get_remote_data_command(client, args):
+    parsed_args = GetRemoteDataArgs(args)
+    try:
+        new_incident_data: Dict = client.retrieve_a_specific_incident_details_request(
+            parsed_args.remote_incident_id
+        )
+
+        parsed_entries = []
+
+        new_incident_data["lumu_incidentId"] = new_incident_data["id"]
+        new_incident_data["severity"] = 1
```

Comment:
why you set the severity to 1 hardcoded?

Reply:
removed harcoded severity in commit 2bb633df7de2b53b93d8a8cdc1fcaef13e33c6a7

Reply:
you can gets the severity value from the incident if you want